tests/unittests/graphOptzTest.cpp

```diff
@@ -665,3 +665,23 @@ TEST_F(GraphOptz, FuseRescaleIntoArithmetic) {
   EXPECT_EQ(mul->getNthInput(0).getType(), rescaleOutTy);
   EXPECT_EQ(div->getNthInput(0).getType(), rescaleOutTy);
 }
+
+TEST_F(GraphOptz, sinkRescaledQuantizedNode) {
+  // Eliminate rescales.
```

Comment:
// Check that we eliminate rescale nodes by sinking them into other operators.

Reply:
fixed, thanks.